test/test_max_args.cpp

```diff
@@ -12,6 +12,8 @@
 #include <boost/test/unit_test.hpp>
 
 #define IDENTITY(z, n, d) d
+// Number of arguments is (now) unlimitted, so take any high value here
+#define MOCK_MAX_ARGS 100
```

Comment:
This is now unneeded to define this, isn’t it?

Reply:
It is used below to generate functions using that many args. Will rename the define.